v8/accordion/accordion.py

```diff
@@ -30,7 +32,18 @@ def set_site(self, site):
 class Accordion(Directive):
     """reST extension for inserting accordions."""
 
+    def __init__(self, name, arguments, options, content, lineno,
+                 content_offset, block_text, state, state_machine):
+        super().__init__(name, arguments, options, content, lineno,
+                         content_offset, block_text, state, state_machine)
+        self.state_id = id(state)
+
     has_content = True
+    optional_arguments = 1
+    # the purpose of this counter stuff is to give each Accordion on a page a
+    # unique number starting with one. These numbers become a part of the
+    # Accordion's html id and aria controls via the template.
+    counters = defaultdict(count)
```

Comment:
This won’t work, please restore the UUIDs. You will have duplicate numbers if two different posts with accordions are on one page (with indexes) and they are built in separate `nikola build` runs.

Reply:
Understood. That's a good point, I didn't realize I was introducing a problem by changing that. Thank you for the feedback.

I just restored the UUIDs.